generators/server/templates/src/main/java/package/_Application.java

```diff
@@ -54,6 +55,8 @@
     @Inject
     private Environment env;
 
+    public static ApplicationContext context;
```

Comment:
I'm not totally sure we should have the app context in a static variable. For what we currently do, of course there is no issue, but the original reason of not having "true" singletons in Spring (which would be static) is for running several app context in parallel in the same JVM. So maybe that could cause issue for some people. Like if they run several JHipster war in an app server, with a "strange" classloader (like JBoss - which I consider as totally wrong), there is a high risk that their app contexts get mixed up.